text/0000-windows-security-identifiers.md

```diff
@@ -0,0 +1,257 @@
+
+For context, file and directory entries in Microsoft Windows NTFS file systems use a Security Descriptor encoded into the file system to store owner/group data. Windows OCI layers also encode a Security Descriptor in the tar header to store owner/group data for container files. The Security Descriptor is a data structure comprised of a pair of Security Identifier (SID) strings to specify a file's Owner and Group (along with optional Access Control Lists for detailed access permissions). This differs from POSIX file systems which use User Identifier/Group Identifier integer values encoded into the file system/OCI layer tar header.
+
+Up until now, CNB platforms and implementations either ignore or implicitly convert POSIX UIDs/GIDs to Windows SIDs to create appropriately permissioned files in image layers. This RFC proposes changing the implementations/platforms (via the spec) so all existing references to a POSIX *UID/GID* will become a Windows *Owner SID/Group SID* for Windows images, environment variables, and file/directory entries in layers and volumes.
+
+There are various technical details involved with this design change, detailed below.  
+
+# Definitions
+[definitions]: #definitions
+* *Security Identifier*: A unique value of variable length that is used to identify a Windows *Security Principal*. Expressed as a string. Maximum 256 bytes.
+  * [Microsoft SID definition](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/78eb9013-1c3a-4970-ad1f-2b1dad588a25) 
+  * [SID components](https://docs.microsoft.com/en-us/windows/win32/secauthz/sid-components)
+  * [SID formal spec](https://docs.microsoft.com/en-us/windows/win32/secauthz/security-identifiers)
+  * [Well-known Windows SIDs](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/81d92bba-d22b-4a8c-908a-554ab29148ab)
+  * [Container-specific SIDs in moby/moby](https://github.com/moby/moby/blob/751d5f879a4f625bd32b08481bbb45e5d2db5b6c/pkg/system/syscall_windows.go)
+* *Security Principal*: Any entity that can be authenticated by the Windows operating system, such as a user account, a computer account, or a thread or process that runs in the security context of a user or computer account, or the security groups for these accounts
```

Comment:
I would put this definition before the definition of "Security Identifier" because "Security Principal" is part of the "Security Identifier" definition.